spec/features/user_spec.rb

```diff
@@ -48,3 +38,33 @@
     expect(page).to have_content('email2@email.com')
   end
 end
+
+feature 'User can log out' do
+  let(:user) { create_logged_in_user }
+  scenario 'clicks logout' do
+    visit root_path(user)
+    click_link 'Logout'
+    expect(page).to have_content('Password')
+  end
+end
+feature 'User can\'t see a list of users' do
+  let(:user) { create_logged_in_user }
+  scenario 'user only see logout menu not User List' do
+    visit root_path(user)
+    expect(page).to_not have_content('User List')
+    expect(page).to have_content('Logout')
+  end
+end
+
+feature 'Only admin can see users' do
+  let(:user) { create_logged_in_user }
+  let(:admin) { create_logged_in_admin }
+  scenario 'user see flash message from user url' do
+    visit users_path(user)
+    expect(page).to have_content('You are not an admin')
+  end
+  scenario 'admin can not see flash message from user url' do
+    visit users_path(admin)
+    expect(page).to_not have_content('You are not an admin')
```

Comment:
Is there some content that we _can_ expect to see on this page if the logged-in user is an admin?

Reply:
Add a test for admin case